auth-jwt-service/src/main/java/io/stargate/auth/jwt/AuthzJwtService.java

```diff
@@ -152,6 +152,64 @@ public void authorizeSchemaWrite(String token, String keyspace, String table, Sc
     // Cannot perform authorization with a JWT token so just return
   }
 
+  /**
+   * Authorization for role management is not provided by JWTs so all authorization will be deferred
+   * to the underlying permissions assigned to the role the JWT maps to.
+   *
+   * <p>{@inheritdoc}
+   */
+  @Override
+  public void authorizeRoleManagement(String token, String role, Scope scope)
+      throws UnauthorizedException {
+    // Cannot perform authorization with a JWT token so just return
```

Comment:
maybe add `log.debug()`? It maybe useful for debugging auth

Reply:
Should we deny access in this case?

Reply:
Access here would depend on the permissions of the underlying role associated with the JWT.

Reply:
It's provided in the interface in the event you want to implement your own auth-provider that does handle authz like C* is doing.